specify/specify7
# Add a tool to create root node in trees

Fixes #5391

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- open the taxon tree viewer 
- create a new tree
- [ ] verify there is now a + button where the root node is usually 
- click the + 
- [ ] verify a root node is created (This might take a few seconds, and then it will reload.) 
- add a child to the new root
